litex/tools/litex_term.py

```diff
@@ -97,6 +97,10 @@ def __init__(self, name="uart_xover", host="localhost", base_address=0): # FIXME
         if not present:
             raise ValueError(f"CrossoverUART {name} not present in design.")
 
+        # On PCIe designs, CSR is remapped to 0 to limit BAR0 size.
```

Comment:
I'm not sure what the default for this should be. Technically, it's also entirely possible for a uart to wishbone bridge to exist as well as a PCIe phy (maybe without CSR mapping). It's also possible someone changed the size of the BAR so that the entire memory map is visible via a BAR.